### PR TITLE
Fix 363

### LINKED ISSUE #363: The app uses username instead of display name
**Describe the bug**
When a citizens name has to be displayed in the app, the username is displayed instead of the display name. The display name is not required when creating a new user, which might be the problem.

**To Reproduce**
1. Setup a local database
2. Make sure you set the weekplaner to use this database
3. Make sure the feature described in issue #66 is implemented
4. Use this new feature to create a new citizen
4.1. Display name and username should be noticeable different
5. Find the new user and see which name is used in the app

**Expected behavior**
When a users name is displayed I expect the app to use the display name

**Actual behavior**
The app uses the username when the display name should be used

**Screenshots**
createCitizen
createCitizen_weekplan

**Environment (please complete the following information):**
- OS: Android
- Emulator: Tested on emulator
- Phone/Tablet: Nexus 7
- APK Version [e.g. 22]

**Additional context**

### PR DESCRIPTION
**Pull request by T-Hauge**
Fix #363
In combination with the following PR:
https://github.com/aau-giraf/api_client/pull/48
https://github.com/aau-giraf/web-api/pull/110
https://github.com/aau-giraf/wiki/pull/186

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw607f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [x] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw612f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [ ] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [ ] Names are meaningful and self-documenting
- [ ] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [ ] All new functionality is covered by tests
- [ ] It is understandable, by reading the tests, what they do
- [ ] The tests covers sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [ ] The features that the code claims to implement, are actually implemented
- [ ] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

File: test/blocs/new_weekplan_bloc_test.dart [hunk 1]
```diff
@@ -27,8 +27,8 @@ void main() {
       accessLevel: null,
       imageUrl: 'http://any.tld',
       imageHash: null);
-  final UsernameModel mockUser =
-      UsernameModel(name: 'User', id: '1', role: null);
+  final DisplayNameModel mockUser =
+  DisplayNameModel(displayName: 'User' , id: '1', role: null);
```
Comment (Jakobzs) [inline test/blocs/new_weekplan_bloc_test.dart]: Remove space after 'User'

**Review by Jakobzs — CHANGES_REQUESTED** [review]

**Comment by jbjarn17** [comment]
Test regarding linter will fail because it depends on other PR's from web-api and api_client

**Review by Jakobzs — APPROVED** [review]

**Review by MathiasKRasmussen — APPROVED** [review]

**Pull request merged**